recipes/pydrive2/LICENSE

```diff
@@ -0,0 +1,185 @@
+Copyright 2013 Google Inc. All Rights Reserved.
```

Comment:
Can you delete this file? It is picked up from the source tarball

Reply:
@isuruf done!